repos/fdbt-site/src/pages/api/deleteAllProducts.ts

```diff
@@ -0,0 +1,23 @@
+import { NextApiResponse } from 'next';
+import { deleteProductByNocCodeAndId, getAllProductsByNoc } from '../../data/auroradb';
+import { redirectToError, redirectTo, getAndValidateNoc } from '../../utils/apiUtils';
+import { NextApiRequestWithSession } from '../../interfaces';
+import { deleteFromS3 } from '../../data/s3';
+import { PRODUCTS_DATA_BUCKET_NAME } from '../../constants';
+
+export default async (req: NextApiRequestWithSession, res: NextApiResponse): Promise<void> => {
+    try {
+        if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test') {
+            const nationalOperatorCode = getAndValidateNoc(req, res);
+            const products = await getAllProductsByNoc(nationalOperatorCode);
+
+            for (const product of products) {
```

Comment:
could we use a forEach here?

Reply:
https://gist.github.com/joeytwiddle/37d2085425c049629b80956d3c618971
Not recommended to use the async in foreach as per the official documentation also